src/core/mount.c

```diff
@@ -1948,7 +1948,7 @@ static void mount_enumerate(Manager *m) {
                         goto fail;
                 }
 
-                r = sd_event_source_set_ratelimit(m->mount_event_source, 1 * USEC_PER_SEC, 5);
+                r = sd_event_source_set_ratelimit(m->mount_event_source, 1 * USEC_PER_SEC, m->default_mount_rate_limit_burst ?: 5);
```

Comment:
Please drop the ternary op. First of all, you don't need it, the field should be set to 5 already by default anyway. Secondly, you take away the ability to turn off the ratelimiting by this. And I think it makes a ton of sense to be able to allow users to turn it off if they want. Hence, by just dropping the `?: 5` you lose nothing but gain simplicity and usefulness.